### ISSUE TITLE
DESCOPE - (HYPER) (BFA) - Terrified Mule Mount

### ISSUE THREAD

**Issue by Gunnydelight** [Gunnydelight/mozzFarmRoutes#42]
https://www.reddit.com/r/woweconomy/comments/993w7i/area_next_to_waycrest_manor_can_be_aoe_farmed_for/

ITEMS:  Greens, Cloth, Green Cloths, x10 herbing nodes

LIOCATION: Area next to Waycrest Manor can be AOE farmed for BOE epics and Terrified Mule Mount [Not hotfixed yet] West of Waycrest Manor

Terrified Mule MOunt
Greens
Tidespray Linen
Deep Line
350+ BOEs

So this place still has not been hotfixed or nerfed. Went out there with some friends and farmed for a few hours.

Its the camp south west of Waycrest Manor instance. Kill per minute is really good. Some random group of like 15 mobs comes sometimes too. Some event or something. Guessing they will hit this place too with a TooMuchFun hammer some time soon.

TLDR: https://www.youtube.com/watch?v=blbB0mI6D0k
